.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  aws-s3: circleci/aws-s3@3.0.0
```

Comment:
ok, what's going on here? This was already done in https://github.com/mui/mui-toolpad/pull/1413

Reply:
Bad merge of master i guess, let me check

Reply:
Ah, because this commit isn't based on master, I shouldn't have merged like this

Reply:
reverted